examples/with_custom_redis_pool.rs

```diff
@@ -27,20 +27,23 @@ impl OpenStatusFromRedisModifier {
 impl StatusModifier for OpenStatusFromRedisModifier {
     fn modify(&self, status: &mut api::Status) {
         let mut conn = self.pool.get().unwrap();
-        let state: RedisResult<String> = conn.get("state_open");
-        status.state.open = match state {
-            Ok(v) => Some(v == "open"),
-            Err(_) => None,
-        };
-        status.state.lastchange = conn.get("state_lastchange").ok();
-        status.state.trigger_person = conn.get("state_triggerperson").ok();
+        let redis_state: RedisResult<String> = conn.get("state_open");
+        if let Some(state) = &mut status.state {
+            state.open = match redis_state {
+                Ok(v) => Some(v == "open"),
+                // v0.13 doesn't allow to remove the state.open field, so we just default to false
```

Comment:
is the 'v0.13' a typo? Anyway, I think removing 'None' as the status is a bit lossy, what should the spaces that don't monitor status put? Should they stay in v13?

Reply:
We want to be compatible with v0.13 and v14 by default. But we fixed this issue in https://github.com/SpaceApi/schema/pull/91 and I'll update spaceapi-rs and spaceapi-server-rs accordingly to keep the old behavior.

Reply:
See https://github.com/spaceapi-community/spaceapi-rs/pull/94 for the 0.8.1 release of spaceapi-rs.

Reply:
Done.